kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js

```diff
@@ -81,11 +81,14 @@ define([
                     this.updateView();
                 });
 
-                this.stagingAreaViewer = new StagingAreaViewer(this.$myFiles, {
-                    path: this.path,
-                    updatePathFn: this.updatePath.bind(this),
-                    userInfo: userInfo,
-                });
+                const stagingAreaArgs = Object.assign({},
+                    this.options.stagingAreaViewer || {},
+                    {
+                        path: this.path,
+                        updatePathFn: this.updatePath.bind(this),
+                        userInfo: userInfo
+                    })
```

Comment:
allow the passing-in of options for the stagingAreaViewer -- otherwise there is no way to do so from the dataStagingTab.